sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -246,7 +246,7 @@ def serialize_iso(attr):
 
     :param Datetime attr: Object to be serialized.
     :rtype: str
-    :raises: ValueError if format invalid.
+    :raises ValueError if format invalid.:
```

Comment:
The closing colon should be after ValueError:
`:raises ValueError: If format is invalid.`